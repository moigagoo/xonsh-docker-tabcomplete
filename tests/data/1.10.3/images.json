{
        "arguments": [
                "-a", "--all",
                "--digests",
                "-f", "--filter=[]",
                "--format",
                "--help",
                "--no-trunc",
                "-q", "--quiet"
        ],
        "commands": []
}
