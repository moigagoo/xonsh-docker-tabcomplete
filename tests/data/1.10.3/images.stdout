
Usage:	docker images [OPTIONS] [REPOSITORY[:TAG]]

List images

  -a, --all          Show all images (default hides intermediate images)
  --digests          Show digests
  -f, --filter=[]    Filter output based on conditions provided
  --format           Pretty-print images using a Go template
  --help             Print usage
  --no-trunc         Don't truncate output
  -q, --quiet        Only show numeric IDs
